web/server/executor/tester_test.go

```diff
@@ -83,13 +83,13 @@ func NewTesterFixture() *TesterFixture {
 	self.shell = NewTimedShell()
 	self.tester = NewConcurrentTester(self.shell)
 	self.packages = []*contract.Package{
-		&contract.Package{Path: "a"},
```

Comment:
all these things are due to gofmt abbreviating stuff. I can remove them from the PR if you prefer, but I think eventually someone will end up committing them anyway :)